app-admin/updateservicectl/updateservicectl-9999.ebuild

```diff
@@ -11,7 +11,7 @@ inherit cros-workon coreos-go
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm64"
 else
-	CROS_WORKON_COMMIT="0b60d85b5ebbe7f097c1c8e19f47fc7307ca322e" # v1.4.0
+	CROS_WORKON_COMMIT="6095da987be1c7423d07992b6b13626b79633043" # v2.0.0
```

Comment:
The tag is on `85923bebbcd0a1964d02fa3d78f50773633ec16d`.  (The SHA-1 here is apparently not a commit.)

Reply:
oh, whoops, that's the sha of the tag, not the sha of the commit (ran `git rev-parse v2.0.0`)